frontend/src/v5/ui/routes/viewer/groups/groupItem/groupItem.component.tsx

```diff
@@ -16,7 +16,7 @@
  */
 import { GroupsActionsDispatchers } from '@/v5/services/actionsDispatchers';
 import { GroupsHooksSelectors } from '@/v5/services/selectorsHooks';
-import LightingIcon from '@assets/icons/lighting.svg';
+import LightingIcon from '@assets/icons/filled/lighting-filled.svg';
```

Comment:
I've just realised this should be **lightning** instead of **lighting**

Reply:
Fixed